### PR TITLE
Add activerecord key to FR locale

### PR DESCRIPTION
**Pull request by fwininger**

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+fr:
+  activemodel: &errors
+    errors:
+      messages:
+        invalid_email_address: 'se semble pas être une adresse email valide'
```
Comment (ngouy) [inline config/locales/fr.yml]: 'se semble pas être une adresse email valide' must be 'ne semble pas être une adresse email valide'

Comment (fwininger) [inline config/locales/fr.yml]: :+1: thank you, I fix it quickly.

**Comment by fwininger** [comment]
Done!

**Comment by ngouy** [comment]
sadly, the gem seem's no more maintained ...

**Comment by alexdunae** [comment]
Merci! I'm the original creator of this gem and am swinging through to merge in some old PRs.

**Comment by fwininger** [comment]
@alexdunae ouha 😲 I don't remember having done this PR.

**Comment by alexdunae** [comment]
It was a loooooooooong time ago

**Comment by ngouy** [comment]
image

**Pull request merged**